Quartz/Engine/Include/Quartz/Events/IEventListener.hpp

```diff
@@ -31,11 +36,11 @@ namespace qz
 	{
 		class IEventListener
 		{
-		public:
-			IEventListener() = default;
-			virtual ~IEventListener() = default;
+		  public:
+			IEventListener ()          = default;
```

Comment:
Do we want spaces before parentheses?

Reply:
Resolved, space before parens only appears now for control statements.